.golangci.yml

```diff
@@ -61,12 +61,15 @@ linters-settings:
         # Whether to skip (*x).method() calls where x is a pointer receiver.
         # Default: true
         skipRecvDeref: false
+  lll:
+    # Make sure `golines` formatter command in the Makefile matches this.
+    line-length: 100
```

Comment:
I tend to find 120 characters substantially less disruptive than 100 characters (and it would likely result in a vastly smaller diff on the format commit)

Anecdotally, my monitor / font size setup is able to display two files side-by-side with maxlen 120, without needing to scroll (and I use a pretty large font size)

Reply:
[c064fa8](https://github.com/Layr-Labs/eigenda-proxy/pull/331/commits/c064fa82496b3988f55d4b835df7d531385e1da2)